colab/tflite_text_classification.ipynb

```diff
@@ -0,0 +1,514 @@
+        "    label = labels[0]\n",
+        "    confidence = output[0]\n",
+        "  else:\n",
+        "    label = labels[1]\n",
+        "    confidence = output[1]\n",
+        "\n",
+        "  print(\"Label: \" + label + \"\\nConfidence: \" + str(confidence))"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "xq00JknF0cOz",
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "outputId": "32ad1ac7-18b4-43fd-f364-a6e1e1d1dab1"
+      },
+      "source": [
+        "#@title Text samples\n",
+        "positive_text = \"This is the best movie I've seen in recent years. Strongly recommend it!\"\n",
+        "negative_text = \"What a waste of my time.\"\n",
+        "\n",
+        "print(positive_text)\n",
+        "print(tokenize_input(positive_text))"
+      ],
+      "execution_count": null,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "This is the best movie I've seen in recent years. Strongly recommend it!\n",
+            "[[   1   13    8    3  117   19  206  109   10 1134  152 2301  385   11\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0]]\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "fvxhY1W7X7Gf"
+      },
+      "source": [
+        "## TFLite\n",
```

Comment:
To add to scott's comments, I think you can make it clearer that the "TFLIte and "IREE" sections are in essence doing the same thing but with different runtimes. 

In addition to altering th etitles, you can give this section a bigger header to separate the "setup" section from the "runtime section" and follow with a note under the latter that the TFLite and IREE runtimes should generate the same output.